libafl_qemu/Cargo.toml

```diff
@@ -129,6 +129,11 @@ bytes-utils = "0.1"
 typed-builder = "0.19"
 memmap2 = "0.9"
 getset = "0.1"
+bitflags = "2.6.0"
+perf-event-open-sys = "4.0.0" # Uses Linux 5.19.4 headers, consider forking if we need to bump
+caps = "0.5.5" # TODO: Mark deps used only in systemmode as optional (requires fixing linux_build.rs)
+raw-cpuid = "11.1.0"
+libipt = { git = "https://github.com/Marcondiro/libipt-rs" } # v2 is not on crates.io
```

Comment:
The project seems active, can we ask them to release? We can't refer to github if we want to put our stuff on crates.io

Reply:
This dep won't make it to the final PR, I'll have to replace it anyway :)